tiny-postgresql-event-store/src/main/kotlin/jp/veka/PostgresClientEventStore.kt

```diff
@@ -0,0 +1,153 @@
+package jp.veka
+
+import jp.veka.converter.JsonEntityConverter
+import jp.veka.converter.ResultSetToEntityMapper
+import jp.veka.exception.UnknownEntityException
+import jp.veka.executor.ExceptionLoggingSqlQueriesExecutor
+import jp.veka.factory.PostgresConnectionFactory
+import jp.veka.query.Query
+import jp.veka.query.QueryBuilder
+import jp.veka.query.select.SelectQuery
+import jp.veka.tables.ActiveEventStreamReaderDto
+import jp.veka.tables.EventRecordDto
+import jp.veka.tables.EventRecordTable
+import jp.veka.tables.EventStreamActiveReadersTable
+import jp.veka.tables.EventStreamReadIndexDto
+import jp.veka.tables.EventStreamReadIndexTable
+import jp.veka.tables.SnapshotDto
+import jp.veka.tables.SnapshotTable
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import ru.quipy.database.EventStore
+import ru.quipy.domain.ActiveEventStreamReader
+import ru.quipy.domain.EventRecord
+import ru.quipy.domain.EventStreamReadIndex
+import ru.quipy.domain.Snapshot
+import java.sql.ResultSet
+import kotlin.reflect.KClass
+
+class PostgresClientEventStore(
+    private val databaseConnectionFactory: PostgresConnectionFactory,
+    private val eventStoreSchema: String = "event_sourcing_store",
+) : EventStore {
+    private val resultSetToEntityMapper: ResultSetToEntityMapper = ResultSetToEntityMapper(JsonEntityConverter())
+    private val executor: ExceptionLoggingSqlQueriesExecutor = ExceptionLoggingSqlQueriesExecutor(logger)
```

Comment:
also should be injected in constructor. If you wish you can define default values